pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>edge-sip2</artifactId>
-  <version>1.5.0-SNAPSHOT</version>
+  <version>2.0.0</version>
```

Comment:
You may want to leave it as 2.0.0-SNAPSHOT if we were to follow the community's steps to release the module. There will be steps to automatically update the pom file.

Reply:
OK